components/JobForm.js

```diff
@@ -35,7 +35,7 @@ function NewDRAForm({ newJob }) {
     // Needed for dynamic routing
     const [id, setId] = useState('')
 
-    // Gets the entry just made
+    // Gets the entry just made for
     const FindId = async () => {
         try {
             const jobRes = await fetch(`${server}/api/jobs`, {
```

Comment:
@EllairaT I did not see the changes you did on the, so just have a check before accepting the pull request.